lib/src/utils.dart

```diff
@@ -30,6 +30,10 @@ Map/*<K, V>*/ updateMap/*<K, V>*/(
 ///     mapToQuery({"foo": "bar", "baz": "bang"});
 ///     //=> "foo=bar&baz=bang"
 String mapToQuery(Map<String, String> map, {Encoding encoding}) {
+  map.keys
```

Comment:
Create a local variable and do this in two steps. It'll be easier to read.

Also: please add a test that exercises this case.